packages/eslint-config-eslint/default.yml

```diff
@@ -92,6 +99,7 @@ rules:
     space-unary-ops: ["error", {words: true, nonwords: false}]
     spaced-comment: ["error", "always", { exceptions: ["-"]}]
     strict: ["error", "global"]
+    template-curly-spacing: ["error", "never"]
     valid-jsdoc: ["error", {
```

Comment:
I'm happy to use `always` too. I chose `never` because it was the rule default and I think I have seen it used more that way.